homeassistant/components/sensor/yr.py

```diff
@@ -185,7 +187,8 @@ def try_again(err: str):
                 model = self.data['meta']['model']
                 if '@nextrun' not in model:
                     model = model[0]
-                self._nextrun = dt_util.parse_datetime(model['@nextrun'])
+                self._nextrun = dt_util.parse_datetime(model['@nextrun']) +\
```

Comment:
You should not do this, since you will end up not having data.

Nextrun only determines IF we should do the web request, not WHEN to do it see [here](https://github.com/home-assistant/home-assistant/blob/289f10feb192e235384a2015462c79f8a140100a/homeassistant/components/sensor/yr.py#L163)